app/main/views/buyers.py

```diff
@@ -0,0 +1,33 @@
+from flask import render_template, request, flash
+from flask_login import login_required
+
+from .. import main
+from ... import data_api_client
+from ..auth import role_required
+
+
+@main.route('/buyers', methods=['GET'])
+@login_required
+@role_required('admin')
+def find_buyer_by_opportunity_id():
```

Comment:
It's fine to have `opportunity_id` in the public-facing URL, but for the rest of the code it should be `brief` to be consistent with all of the other code everywhere else where they are called briefs.

e.g. `brief_id = request.args.get('opportunity_id')`  and then `brief_id` from then on.